src/sentry/eventstream/kafka/postprocessworker.py

```diff
@@ -75,20 +76,24 @@ def dispatch_post_process_group_task(
     primary_hash: Optional[str],
     skip_consume: bool = False,
     group_states: Optional[GroupStates] = None,
+    group_category: Optional[GroupCategory] = GroupCategory.ERROR,
 ) -> None:
     if skip_consume:
         logger.info("post_process.skip.raw_event", extra={"event_id": event_id})
     else:
         cache_key = cache_key_for_event({"project": project_id, "event_id": event_id})
 
-        post_process_group.delay(
-            is_new=is_new,
-            is_regression=is_regression,
-            is_new_group_environment=is_new_group_environment,
-            primary_hash=primary_hash,
-            cache_key=cache_key,
-            group_id=group_id,
-            group_states=group_states,
+        post_process_group.apply_async(
+            kwargs={
+                "is_new": is_new,
+                "is_regression": is_regression,
+                "is_new_group_environment": is_new_group_environment,
+                "primary_hash": primary_hash,
+                "cache_key": cache_key,
+                "group_id": group_id,
+                "group_states": group_states,
+            },
+            queue=get_post_process_queue(group_category),
```

Comment:
don't totally understand the difference between this call and the other one. when does each get used?

Reply:
This call is used by the real Kafka post process worker, which is used in production. The other is used by `SnubaEventStream`, which just dispatches the task on eventstream insert instead.

Reply:
I believe this one isn't really used in prod. Not sure about the history but I can imagine this was maybe a reference implementation and the Snuba and/or Kafka ones came later and superceeded the base EventStream impl.

Reply:
Is `SnubaEventStream` only used in dev stuff to avoid running kafka and zookeeper?

Reply:
Yes, it's primarily used in dev